common/log/rap_logs.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+import time
+from concurrent.futures import ThreadPoolExecutor
+from os import name
+from typing import Optional
+
+from requests import RequestException
+
+import settings
+from common.log import logger
+from common.ripple import userUtils
+from common.web.discord import Webhook
+from objects import glob
+
+THREAD_POOL = ThreadPoolExecutor(max_workers=4)
+
+ENDL = "\n" if name == "posix" else "\r\n"
+
+RETRY_INTERVAL = 1
+MAX_RETRIES = 3

```

Comment:
now that the firing of discord webhooks is done in the background, we can enforce a much more aggressive retry policy
```suggestion
RETRY_INTERVAL = 8
MAX_RETRIES = 10
```